src/components/table/table.tsx

```diff
@@ -263,8 +263,12 @@ export class Table {
     }
 
     @Watch('data')
-    protected updateData(newData = [], oldData = []) {
-        if (isEqual(newData, oldData)) {
+    protected updateData(newData: RowData[] = [], oldData: RowData[] = []) {
```

Comment:
This commit makes the sorting buggy when you press a column name. I guess we can't use the `updateOrAddData` since it has no new rows in the case of re-ordering ASC/DESC.

Reply:
See https://github.com/Lundalogik/crm-feature/issues/4247 for bug